input/assets/js/main.js

```diff
@@ -498,3 +498,18 @@ $(function() {
     $("#my-account-desktop-content").insertAfter("#my-account-mobile-content");
   }
 });
+
+/*
+My Account: Change password
+*/
+
+$('.password-feedback-wrapper').hide();
+$('.password-change-btn').click(function() {
+  $('.change-password-wrapper').hide();
+  $('.password-feedback-wrapper').fadeIn();
+})
+
+$('.password-back-btn').click(function() {
+  $('.password-feedback-wrapper').hide();
+  $('.change-password-wrapper').fadeIn();
+})
```

Comment:
This same behaviour needs to go to my personal details (since it's actually not working, as you might have realized), so I would recommend you to extract the common behaviour to avoid repetition.

To solve that problem, in my own code I made something like this:
```
// Initializes editable data
function openForm(formClassName) {
    $("." + formClassName + "-hide").hide();
    $("." + formClassName + "-show").fadeIn();
}

function closeForm(formClassName) {
    $("." + formClassName + "-hide").fadeIn();
    $("." + formClassName + "-show").hide();
}

function initializeEditableData(formClassName) {
   var formWrapper = $(formClassName);
   if (formWrapper.hasClass("in")) {
       openForm(formClassName);
   } else {
       closeForm(formClassName);
   }
   $("." + formClassName + "-show-btn").click(function(){ openForm(formClassName); });
   $("." + formClassName + "-hide-btn").click(function(){ closeForm(formClassName); });
}

initializeEditableData("personal-details-edit");
```

This required to change the name of some classes on the way. Here is my own template for my personal details (the one I use in real Sunrise): https://github.com/commercetools/commercetools-sunrise-java/blob/master/conf/templates/my-account-personal-details.hbs